inc/yhtion_parametritrivi.inc

```diff
@@ -2980,12 +2980,27 @@ if ($fieldname == "livetuotehaku_poistetut") {
 
   $ulos .= "<option value = ''>".t("Poistuvia/poistettuja tuotteita ei n�ytet�")."</option>";
   $ulos .= "<option value = 'X' $sel[X]>".t("Poistuvat tuotteet n�ytet��n")."</option>";
+  $ulos .= "<option value = 'P' $sel[P]>".t("Poistuvat ja poistetut tuotteet n�ytet��n")."</option>";
```

Comment:
@tiere Tarkoittaako tämä, että KAIKKI tuotteet näytetään? Jos näin on, niin voisi olla selkeämpää kirjoittaa, että "Näytetään kaikki tuotteet". Ehkä?